Application/EdFi.Ods.Api/Security/Extensions/EdFiAuthorizationContextExtensions.cs

```diff
@@ -12,16 +12,33 @@ public static class EdFiAuthorizationContextExtensions
     /// <summary>
     /// Gets one of two possible strings supplied corresponding to the authorization phase of the authorization context.
     /// </summary>
-    /// <param name="authorizationContext">The authorization context which identifies the current authorization phase.</param>
+    /// <param name="dataManagementRequestContext">The authorization context which identifies the current authorization phase.</param>
     /// <param name="existingDataText">The text to return if the phase is <see cref="AuthorizationPhase.ExistingData"/></param>
     /// <param name="proposedDataText">The text to return if the phase is <see cref="AuthorizationPhase.ProposedData"/></param>
     /// <returns>The appropriate supplied string.</returns>
     public static string GetPhaseText(
-        this EdFiAuthorizationContext authorizationContext,
+        this DataManagementRequestContext dataManagementRequestContext,
```

Comment:
This class can be renamed EdFiDataManagementRequestContextExtensions.

Reply:
I consolidated the usages to eliminate one of the methods and have renamed this class and file to `AuthorizationPhaseExtensions`.